packages/edit-post/src/components/secondary-sidebar/style.scss

```diff
@@ -57,7 +57,7 @@
 .edit-post-editor__list-view-container > .document-outline,
 .edit-post-editor__list-view-empty-headings {
 	overflow-x: hidden;
-	overflow-y: auto;
+	overflow-y: scroll;
```

Comment:
In my testing I have found that "auto" will trigger repaint of every children when elements are added/removed from the DOM in a parent, which is what happens with the popovers. Using scroll seems to prevent the repaint of all the children. It does maintain the scrollbat even when it's not "needed"

Reply:
I have reverted the changes on the skeleton (overflow and sidebar width)